.github/workflows/secure-deployment.yml

```
@@ -306,6 +306,7 @@ jobs:
       with:
         status: ${{ job.status }}
         channel: '#deployments'
+        webhook_url: ${{ secrets.SLACK_WEBHOOK }}
```

actionlint (1.7.11)

[error] input "webhook_url" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "cancelled_message", "channel", "custom_payload", "failure_message", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "success_message", "text", "username"

(action)